autosfw2jpg/autosfw2jpg

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+ROLL_NUMBER=$1
+PICTURE_DATE=$2
+
+BLOCK_DEVICE=/dev/sr0
```

Comment:
Let's put a comment here to help the user know where to update:

# Settings - Update these to match your system before running